### PR TITLE
Add link to other examples.

### PR DESCRIPTION
**Pull request by tttza**
Some users have found it difficult to understand how to describe multiple domains in a policy setting.
I added an explanation to the Edge policy documentation to guide users to other examples.

### CODE REVIEW

File: SharePoint/SharePointOnline/sharepoint-view-in-edge.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Follow the steps below to use View in File Explorer in Microsoft Edge:
 
         :::image type="content" source="media/edgepolicy-adeejoseph.png" alt-text="Enable Configure the View in File Explorer feature for SharePoint pages in Microsoft Edge":::
 
+        For more examples, see [ConfigureViewInFileExplorer](/deployedge/microsoft-edge-policies#configureviewinfileexplorer) on Edge policy reference.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-view-in-edge.md:63]:
```suggestion
        For more examples, see [ConfigureViewInFileExplorer](/deployedge/microsoft-edge-policies#configureviewinfileexplorer) on Microsoft Edge policy reference.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tttza, kindly commit the suggested change. Many thanks.

File: SharePoint/SharePointOnline/sharepoint-view-in-edge.md (hunk 1 above)
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-view-in-edge.md:63]:
```suggestion
        For more examples, see [ConfigureViewInFileExplorer](/deployedge/microsoft-edge-policies#configureviewinfileexplorer) on Microsoft Edge policy reference.
```

**Comment by tttza** [comment]
Thanks for the review.
I forgot to write the official name. 
Committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**